README_zh-CN.md

```diff
@@ -126,79 +106,123 @@ mim install -e .
-- [x] [Transformer-in-Transformer](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/tnt)
-- [x] [Res2Net](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/res2net)
-- [x] [MLP-Mixer](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/mlp_mixer)
-- [x] [DeiT](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/deit)
-- [x] [DeiT-3](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/deit3)
-- [x] [Conformer](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/conformer)
-- [x] [T2T-ViT](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/t2t_vit)
-- [x] [Twins](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/twins)
-- [x] [EfficientNet](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/efficientnet)
-- [x] [EdgeNeXt](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/edgenext)
-- [x] [ConvNeXt](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/convnext)
-- [x] [HRNet](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/hrnet)
-- [x] [VAN](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/van)
-- [x] [ConvMixer](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/convmixer)
-- [x] [CSPNet](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/cspnet)
-- [x] [PoolFormer](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/poolformer)
-- [x] [Inception V3](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/inception_v3)
-- [x] [MobileOne](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/mobileone)
-- [x] [EfficientFormer](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/efficientformer)
-- [x] [MViT](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/mvit)
-- [x] [HorNet](https://github.com/open-mmlab/mmclassification/tree/master/configs/hornet)
-- [x] [MobileViT](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/mobilevit)
-- [x] [DaViT](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/davit)
-- [x] [RepLKNet](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/replknet)
-- [x] [BEiT](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/beit) / [BEiT v2](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/beitv2)
-- [x] [EVA](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/eva)
-- [x] [MixMIM](https://github.com/open-mmlab/mmclassification/tree/1.x/configs/mixmim)
-
-</details>
+相关结果和模型可在 [模型库](https://mmclassification.readthedocs.io/zh_CN/1.x/modelzoo_statistics.html) 中获得。
+
+<div align="center">
+  <b>主要架构</b>
+</div>
+<table align="center">
+  <tbody>
+    <tr align="center" valign="bottom">
+      <td>
+        <b>支持的主干网络</b>
+      </td>
+      <td>
+        <b>自监督学习</b>
+      </td>
+      <td>
+        <b>图像检索</b>
```

Comment:
I task here image retrieval should be removed, as pre-train 3rd package, image retrieval is a downstream function currently pre-trained or classified. We could add a column for others and put some training tips—for example ArcFace, RandAug, AutoAug, RepeatAugSamper, etc.